src/save-cache.ts

```diff
@@ -5,7 +5,7 @@ import {
   STATE_CACHE_MATCHED_KEY,
   STATE_CACHE_KEY,
 } from "./cache/restore-cache";
-import { cacheLocalPath, enableCache } from "./utils/inputs";
+import { cacheLocalPath, enableCache, pruneCache as pruneCacheOption } from "./utils/inputs";
```

Comment:
I would rather name it `shouldPruneCache`. But this is just a personal preference 😉